src/app/EventManagement.cpp

```diff
@@ -584,6 +592,12 @@ static bool IsInterestedEventPaths(EventLoadOutContext * eventLoadOutContext, co
     {
         return false;
     }
+
+    if (!event.mFabricIndex.IsNull() && eventLoadOutContext->mFabricIndex != event.mFabricIndex.Value())
```

Comment:
OK, so here is presumably where we don't want to just use a 0 fabric index to mean "not fabric associated at all"?  Or do we not?  For an event that may or may not be fabric-scoped, how does one indicate "not associated with a particular fabric" in the spec?

Reply:
Yes, since we have nullable fabric index, and in door lock cluster spec, it says if fabric index is null, the generated code is using DataModel::Nullable<FabricIndex>, I prefer use nullable to represent "not fabric associated at all". https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/686ca55b4764cc6e1fccd348671a7360a5f6828e/src/app_clusters/DoorLock.adoc#534-fabricindex

Reply:
Per offline chat, we would check for a field that is of type fabric_idx and not nullable which could be used for fabric-scoped event, currently acl is fabric scope one. updated the code